openapi-communication-service.yaml

```diff
@@ -0,0 +1,714 @@
+          examples:
+            400 BadRequest example:
+              value:
+                errors:
+                  - code: L01
+                    title: Wrongly formated payload
+                    status: 400
+                    detail: Wrong content payload
+                    instance: /communications/6090146f-1023-4f7c-8eed-10a045d37ae8
+        '401':
+          $ref: '#/components/responses/Denied'
+          examples:
+            401 Denied example:
+              value:
+                errors:
+                  - code: 72D
+                    title: Not authorized
+                    status: 401
+                    detail: Authentication credentials were not provided or are invalid
+                    instance: /communications/6090146f-1023-4f7c-8eed-10a045d37ae8
+        '403':
+          $ref: '#/components/responses/Forbidden'
+          examples:
+            403 Forbidden example:
+              value:
+                errors:
+                  - code: 12L
+                    title: Forbidden
+                    status: 403
+                    detail: Privileges of the token are insufficient to perform this action
+                    instance: /communications/6090146f-1023-4f7c-8eed-10a045d37ae8
+        '405':
+          $ref: '#/components/responses/MethodNotAllowed'
+          405 MethodNotAllowed example:
+            value:
+              errors:
+                - code: 42R
+                  title: Method Not Allowed
+                  status: 405
+                  detail: Method not allowed for the specified resource
+                  instance: /communications/6090146f-1023-4f7c-8eed-10a045d37ae8
+        '413':
+          $ref: '#/components/responses/RequestTooLarge'
+          413 RequestToLarge example:
+            value:
+              errors:
+                - code: 42R
+                  title: Request Entity Too Large
+                  status: 413
+                  detail: The payload for the body of the request is too large
+                  instance: /communications/6090146f-1023-4f7c-8eed-10a045d37ae8
+        '429':
+          $ref: '#/components/responses/TooManyRequests'
+          examples:
+            429 TooManyRequests example:
+              value:
+                errors:
+                  - code: 13L
+                    title: Too many requests
+                    status: 429
+                    detail: You reached your request limits
+                    instance: /communications/6090146f-1023-4f7c-8eed-10a045d37ae8
+        default:
+          description: Unexpected error
+          headers:
+            Content-Type:
+              schema:
+                type: string
+                example: application/problem+json
+              description: Indicates the original media type of the resource
+          content:
+            application/problem+json:
+              schema:
+                $ref: ./models/Problems.yaml
```

Comment:
Since we change the approach, can you change these local references to direct URL references?